README.md

```diff
@@ -1,8 +1,14 @@
 Main Branch Status: ![CI/CD Workflow](https://github.com/Health-Education-England/tis-trainee-forms/workflows/CI/CD%20Workflow/badge.svg?branch=main)  
 Deployment Status: ![CI/CD Workflow](https://github.com/Health-Education-England/tis-trainee-forms/workflows/CI/CD%20Workflow/badge.svg?branch=main&event=deployment_status)
-# TIS Trainee Forms
+# TIS Trainee Forms Service
 
```

Comment:
For consistency with the other projects, maybe rework the badging: there's an example here - https://github.com/Health-Education-England/tis-trainee-reference/commit/3ac8e17971c0dbeeafba01e92269bc7bde11545f